scanpy/preprocessing/_highly_variable_genes.py

```diff
@@ -300,6 +303,7 @@ def highly_variable_genes(
     subset: bool = False,
     inplace: bool = True,
     batch_key: Optional[str] = None,
+    check_values: Optional[bool] = True,
```

Comment:
Should this just be `bool`?

```suggestion
    check_values: bool = True,
```